### ISSUE TITLE
Request for release notes and some git tags

### ISSUE THREAD

**Issue by naveensrinivasan** [GoogleContainerTools/distroless#980]
Thanks for this project! We appreciate and use it a lot! One of the issues I am running into is that there aren't any release notes on new releases. 

Here is an example 

https://github.com/ossf/scorecard/pull/1745
![image](https://user-images.githubusercontent.com/172697/158856245-dd58b109-e4f6-48de-b37f-e9a1c277b1ca.png)

- I can't make out as to what changed and why should the project upgrade to the latest release 

Almost all the projects publish release notes https://github.com/ossf/scorecard/pull/1742 which helps in identifying the reason to upgrade a release. 

It would be nice to have release tags and notes.

**Comment by justaugustus** [comment]
cc: @dlorenc @imjasonh @dekkagaijin

**Comment by imjasonh** [comment]
The cc train rolls on with a

cc @loosebazooka

**Comment by loosebazooka** [comment]
Yeah. Uh. It's almost always operating system upgrades. But we'd probably be able to automate some generation of release notes to include in the image (or an sbom that you could diff). It's just not highest priority right now.

Also, distroless just continuously publishes updates. So there's are no release tags. I think if we separate out the build logic and the content config that might be something we could do for the build logic. But it's currently all mashed up together.
